vnext/ReactUWP/Views/Impl/SnapPointManagingContentControl.h

```diff
@@ -1,7 +1,9 @@
 #pragma once
 
-#include "winrt/Windows.Foundation.Collections.h"
-#include "winrt/Windows.Foundation.h"
+#include <UI.Xaml.Controls.Primitives.h>
+#include <UI.Xaml.Controls.h>
+#include <winrt/Windows.Foundation.Collections.h>
+#include <winrt/Windows.Foundation.h>
```

Comment:
These must be in pch.h along with other base C++/WinRT classes.

Reply:
Could you elaborate? Do you mean foundation specifically? That already is, but I didn't remove the existing usage here.